systemtest/intake.go

```diff
@@ -53,3 +59,11 @@ func sendEventsPayload(t *testing.T, srv *apmservertest.Server, urlPath, payload
 	require.NoError(t, err)
 	require.Equal(t, http.StatusAccepted, resp.StatusCode, string(respBody))
 }
+
+func openFile(t *testing.T, p string) *os.File {
```

Comment:
perhaps add a t.Cleanup which closes the file? then we don't need to close it in each test